.goreleaser.yml

```diff
@@ -74,75 +79,11 @@ dockers:
   - "--build-arg=kopia_repo_org=kopia"
   extra_files:
   - 'LICENSE'
-- ids:
-  - kando
-  image_templates:
-  - 'ghcr.io/kanisterio/postgres-kanister-tools:{{ .Tag }}'
-  dockerfile: 'docker/postgres-kanister-tools/Dockerfile'
-  build_flag_templates:
-  - "--build-arg=TOOLS_IMAGE=ghcr.io/kanisterio/kanister-tools:{{ .Tag }}"
-- image_templates:
-  - 'ghcr.io/kanisterio/postgresql:{{ .Tag }}'
-  dockerfile: 'docker/postgresql/Dockerfile'
-  build_flag_templates:
-  - "--pull"
-- ids:
-  - kando
-  image_templates:
-  - 'ghcr.io/kanisterio/es-sidecar:{{ .Tag }}'
-  dockerfile: 'docker/kanister-elasticsearch/image/Dockerfile'
-  build_flag_templates:
-  - "--build-arg=TOOLS_IMAGE=ghcr.io/kanisterio/kanister-tools:{{ .Tag }}"
-- ids:
-  - kando
-  image_templates:
-  - 'ghcr.io/kanisterio/mysql-sidecar:{{ .Tag }}'
-  dockerfile: 'docker/kanister-mysql/image/Dockerfile'
-  build_flag_templates:
-  - "--build-arg=TOOLS_IMAGE=ghcr.io/kanisterio/kanister-tools:{{ .Tag }}"
 - image_templates:
   - 'ghcr.io/kanisterio/kanister-kubectl-1.18:{{ .Tag }}'
   dockerfile: 'docker/kanister-kubectl/Dockerfile'
   build_flag_templates:
   - "--build-arg=TOOLS_IMAGE=ghcr.io/kanisterio/kanister-tools:{{ .Tag }}"
```

Comment:
have we decided not to build these removed images or the build process is moved somewhere else?

Reply:
It's moved to .github/workflows/build_example_images.yaml